services/workflows-service/src/collection-flow/controllers/collection-flow.controller.ts

```diff
@@ -27,8 +27,8 @@ export class ColectionFlowController {
   }
 
   @common.Get('/user')
-  async getUser(@TokenScope() tokenScope: ITokenScope) {
-    return this.service.getUser(tokenScope.endUserId, tokenScope.projectId);
+  async getUser() {
+    return {}; // @TODO: Make sure the client is not using this endpoint and remove it.
   }
```

Comment:
Need to check in which places the `kyb-app` is using this endpoint and make sure it's no longer in usage. Illia says it's not necessary anymore in the `kyb-app`: https://teamballerine.slack.com/archives/C0510CD1R8T/p1696762753574889